package.json

```diff
@@ -4,6 +4,10 @@
   "description": "Check dependencies in your node module",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "bin/*.js",
+    "dist/**/*.{js,json,ts}"
```

Comment:
This could probably be just `"dist/"`, but I don't know if you clean up the dist dir or if you publish the package on CI. If that's the case, `"dist/"` would work too.